plugins/askrene/algorithm.c

```diff
@@ -212,11 +212,35 @@ static s64 get_augmenting_flow(const struct graph *graph,
 	return flow;
 }
 
+
+/* Helper.
+ * Sends an amount of flow through an arc, changing the flow balance of the
+ * nodes connected by the arc and the [residual] capacity of the arc and its
+ * dual. */
+static inline void sendflow(const struct graph *graph, const struct arc arc,
```

Comment:
inline is generally not a good idea in C files.  In fact, it's almost certainly ignored.

Reply:
d24e60a643e377a91163afb0271d42958b04d027